### PR TITLE
feat(toolbar): Add CTA button

### PR DESCRIPTION
**Pull request by charlescatta**
Screen Shot 2021-11-11 at 10 17 25 AM

### CODE REVIEW

**Comment by allardy** [comment]
What is this for .. ??

**Comment by charlescatta** [comment]
> What is this for .. ??

https://linear.app/botpress/issue/BUS-104/add-sales-cta-in-studio

**Comment by charlescatta** [comment]
Fixed the styling
localhost_3000_studio_test_flows_main
g

File: packages/studio-ui/src/web/components/Layout/Toolbar/index.tsx [hunk 1]
```diff
@@ -27,53 +27,60 @@ const Toolbar: FC<Props> = props => {
   return (
     <header className={style.toolbar}>
       <div className={style.list}>
-        {!!hasDoc && (
-          <Fragment>
+        <Tooltip position="right-bottom" content={lang.tr('toolbar.salesCallToActionDescription')}>
```
Comment (laurentlp) [inline packages/studio-ui/src/web/components/Layout/Toolbar/index.tsx:30]:
```suggestion
        <Tooltip position="right-bottom" content={lang.tr('studio.flow.toolbar.salesCallToActionDescription')}>
```

**Review by laurentlp — COMMENTED** [review]
Small corrections to make, otherwise LGTM!

**Comment by charlescatta** [comment]
@laurentlp fixed and tested! thanks for catching that

**Review by laurentlp — APPROVED** [review]

**Pull request merged**